apps/wing/src/commands/figSpec.ts

```diff
@@ -0,0 +1,193 @@
+import { Argument, Command, Option } from "commander";
+import prettier from "prettier";
+
+export interface Options {
+  figSpecCommandName?: string;
+}
+
+const DEFAULT_FIG_SUBCOMMAND_NAME = "generate-fig-spec";
+
+function convertDefaultValue(v: unknown) {
+  if (typeof v === "string") return v;
+  if (Array.isArray(v)) return v.join(",");
+  return String(v);
+}
+
+function getTemplate(spec: Fig.Spec): string {
+  return prettier.format(
```

Comment:
Tested this without prettier, but it looks bad unfortunately 😕
```
// Autogenerated by @fig/complete-commander

    const completionSpec: Fig.Spec = {"name":"wing","subcommands":[{"name":["run","it"],"description":"Runs a Wing program in the Wing Console","options":[{"name":["-p","--port"],"description":"specify port","args":{"name":"port"}},{"name":["--no-open"],"description":"Do not open the Wing Console in the browser"},{"name":["-h","--help"],"description":"display help for command","priority":49}],"args":[{"name":"entrypoint","description":"program .w entrypoint","isOptional":true}]},{"name":"lsp","description":"Run the Wing language server on stdio","options":[{"name":["-h","--help"],"description":"display help for command","priority":49}]},{"name":"compile","description":"Compiles a Wing program","options":[{"name":["-t","--target"],"description":"Target platform","args":{"name":"target","suggestions":["tf-aws","tf-azure","tf-gcp","sim","awscdk"],"default":"sim"}},{"name":["-p","--plugins"],"description":"Compiler plugins","args":{"name":"plugin","isOptional":true,"isVariadic":true}},{"name":["-h","--help"],"description":"display help for command","priority":49}],"args":[{"name":"entrypoint","description":"program .w entrypoint"}]},{"name":"test","description":"Compiles a Wing program and runs all functions with the word 'test' or start with 'test:' in their resource identifiers","options":[{"name":["-t","--target"],"description":"Target platform","args":{"name":"target","suggestions":["tf-aws","tf-azure","tf-gcp","sim","awscdk"],"default":"sim"}},{"name":["-p","--plugins"],"description":"Compiler plugins","args":{"name":"plugin","isOptional":true,"isVariadic":true}},{"name":["-h","--help"],"description":"display help for command","priority":49}],"args":[{"name":"entrypoint","description":"all entrypoints to test","isVariadic":true}]},{"name":"docs","description":"Open the Wing documentation","options":[{"name":["-h","--help"],"description":"display help for command","priority":49}]},{"name":"help","description":"display help for command","priority":49,"args":{"name":"command","isOptional":true,"template":"help"}}],"options":[{"name":["-V","--version"],"description":"output the version number"},{"name":["--debug"],"description":"Enable debug logging (same as DEBUG=1)"},{"name":["--progress"],"description":"Show compilation progress"},{"name":["--no-progress"],"description":"Hide show compilation progress"},{"name":["--no-update-check"],"description":"Skip checking for toolchain updates"},{"name":["--no-analytics"],"description":"Disable analytics collection (same as WING_DISABLE_ANALYTICS=1)"},{"name":["-h","--help"],"description":"display help for command","priority":49}]}

    export default completionSpec;
```

Reply:
Does looking bad matter? I assumed this file is simply used internally, do users actually see this

Reply:
Actually not, but I guess this will be the content of the PR in the fig repo, so it's a matter of how the maintainers will accept this, but however, we can try it. I don't mind, I can remove the `prettier.format` and then lets see how the PR will look like

Reply:
Yeah that would be preferable, it seems pretty wasteful to force the `prettier` dependency for all consumers of the CLI when it's only used for this.